test/ignore_pattern.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var tap = require('tap');
+var path = require('path');
+var execFile = require('child_process').execFile;
+
+var tapeBin = path.join(process.cwd(), 'bin/tape');
+
+tap.test('should allow ignore file together with --ignorePattern', function (tt) {
+	tt.plan(1);
+	var proc = execFile(tapeBin, ['**/*.js', '--ignore', '.ignore', '--ignorePattern', 'fake_other_ignored_dir'], { cwd: path.join(__dirname, 'ignorePattern') });
```

Comment:
i'd expect the file paths to come last, after all the dashed arguments. any reason to order them this way?

Reply:
No particular reason, just changed the order 👌